framework/test/Volo.Abp.MongoDB.Tests/Volo/Abp/MongoDB/Repositories/Repository_Basic_Tests.cs

```diff
@@ -64,5 +64,15 @@ public override async Task InsertAsync()
             person.Phones.Count.ShouldBe(1);
             person.Phones.Any(p => p.PersonId == person.Id && p.Number == "1234567890").ShouldBeTrue();
         }
+
+        [Fact]
+        public async Task Insert_Should_Set_Guid_Id()
```

Comment:
Should't this be tested for EF Core (instead of MongoDB)? Or can we move it to the base `Repository_Basic_Tests<T>`?